homeassistant/components/tibber/sensor.py

```diff
@@ -336,6 +337,11 @@ def device_class(self):
         """Return the device class of the sensor."""
         return self._device_class
 
+    @property
+    def state_class(self) -> str:
+        """Return the state class of this entity, from STATE_CLASSES, if any."""
+        return STATE_CLASS_MEASUREMENT
```

Comment:
Some of the TibberSensorRT measurements are averages or accumulations. I'm not sure that those data fit into this state class category. Maybe we need to distinguish between the sensor types and only mark some of them?

https://developers.home-assistant.io/docs/core/entity/sensor#available-state-classes

@emontnemery what do you think?

Reply:
I thought that is the same with utility_meter?
https://github.com/home-assistant/core/blob/0c40f3733608691e1fe83a1604c0b239e49b402a/homeassistant/components/utility_meter/sensor.py#L334

Reply:
It's a good question.

Reply:
For me, it would make sense to not use STATE_CLASS_MEASUREMENT for min, max and average values, but use it for accumulated values.
What about cost? It is a calculated value based on accumulated consumption and price.

Reply:
Similar use case: https://github.com/home-assistant/core/pull/50978

Reply:
That is not similar, toon assigns it only to sensors that are current values.

From the documentation for the measurement state class:

> The state represents a measurement in present time, not a historical aggregation such as statistics or a prediction of the future.

averages and such may not have this state class set.